app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -1,487 +1,610 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="vertical"
-    app:layout_behavior="@string/appbar_scrolling_view_behavior">
+    android:layout_height="match_parent">
 
-    <RelativeLayout
-        android:id="@+id/display_box"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_alignParentTop="true">
+    <LinearLayout
+        android:id="@+id/linearLayout14"
+        android:layout_width="329dp"
```

Comment:
I'm wondering how we can use this much precise dimensions. Have you tested with different screens? Also these needs to go in `dimens.xml`